content/changelog/2024-12-06.md

```diff
@@ -0,0 +1,107 @@
+
+Stay updated on Drizzle Studio improvements and fixes by following the [Neon Drizzle Studio Integration Changelog](https://github.com/neondatabase/neon-drizzle-studio-changelog/blob/main/CHANGELOG.md).
+
+### A Model Context Protocol (MCP) server for Neon
+
+We’ve released an open-source [Model Context Protocol (MCP)](https://modelcontextprotocol.io/introduction) server for Neon, enabling any MCP Client such as Claude Desktop to interact with Neon’s API using natural language. AI agents can use our MCP server to perform actions such as creating databases, running SQL queries, and managing database migrations. For more, [read the announcement](https://neon.tech/blog/let-claude-manage-your-neon-databases-our-mcp-server-is-here) and try it out on [GitHub](https://github.com/neondatabase/mcp-server-neon).
+
+### Archive storage on paid plans
+
+A few weeks ago, we announced [branch archiving on the Free Plan](/docs/changelog/2024-11-15#branch-archiving-on-the-free-plan), which automatically stores inactive branches in cost-efficient archive storage. This feature is now available on our paid plans: Launch, Scale, and Business. Here’s what this means for you:
+
+- **More storage and reduced storage costs**: Branches **older than 14 days** and **not accessed for 24 hours** are automatically archived in cost-efficient archive storage.
+- **New archive storage allowances by plan**:
+  - Launch: 50 GB
+  - Scale: 250 GB
+  - Business: 2500 GB
+- **Billing for extra archive storage**: Extra storage beyond your plan’s allowance is billed at $0.10 per GB-month. Inactive branches meeting the thresholds above will now cost less.
+- **Monitoring archive storage usage**: A new archive storage metric is available on the **Billing** page in the Neon Console.
+- **Archived branch details and identification**: Archived branches are marked with an archive icon in the console. The **Branches** page shows additional details, including the date and time a branch was archived.
+- **Automatic archiving and unarchiving**: No action is needed to archive or unarchive branches. Unarchiving happens automatically when you access a branch.
+- **Performance considerations**: You should notice little to no performance difference when connecting to or querying an archived branch, except for branches with large amounts of data where initial access may take a little longer until the branch is fully unarchived.
+
+For more details, see [Branch archiving](/docs/guides/branch-archiving) and [Archive storage](/docs/introduction/architecture-overview#archive-storage).
+
+### Simplified "extra storage"
+
+To simplify billing and reduce costs for extra storage beyond your plan's monthly allowance, we’ve switched to billing by GB-month. Previously, if you exceeded your storage allowance, you were allocated (and billed for) an extra storage unit at a prorated price for the rest of the month, even if the overage was brief. Now, you’re billed only for the storage you use.
```

Comment:
How about: 
"We are reducing the cost of temporarily using more storage than is in your plan's monthly allowance. Previously, if you exceeded your storage allowance, you were allocated (and billed for) an extra storage unit at a prorated price for the rest of the month, even if the overage was brief. Now, you’re billed only for the storage you use for the length of time that you use it. This will be measured per GB-month

Reply:
TY. Revised.